### ISSUE TITLE
Missing skip links

### ISSUE THREAD

**Issue by israelcefrin** [pkp/pkp-lib#11338]
## Issue Description
The Dashboard doesn't present any "skip link" option to navigate through the interface using the keyboard, e.g.: skip to the submissions table (jumping the menus).

## How to reproduce / Current behavior
1) Log in to a journal
2) Press the TAB key
3) Notice that you will need to TAB every focusable  element to reach to the submission table

## What application are you using?
 OJS, OMP or OPS version 3.5rc2

## Stack used 
- Device: laptop
- OS: MacOS
- Browser: Edge/Firefox/Chrome/Safari
- Version: latest

## Testing method
- Method:  Manual; Keyboard

## Action
Include a list of links to the diferent sections of the interface that allows users to navigate to the main sections in each interface in the dashboard.

## Additional information
It is the same issue reported in the previous version: https://github.com/pkp/pkp-lib/issues/9312

Success criteria 2.4.1 - Bypass blocks - https://www.w3.org/WAI/WCAG21/Understanding/bypass-blocks

PRs:
**main**
ui-library: https://github.com/pkp/ui-library/pull/663
pkp-lib: https://github.com/pkp/pkp-lib/pull/11640
ojs (tests): https://github.com/pkp/ojs/pull/4993

**Comment by jardakotesovec** [comment]
Hi @israelcefrin, to keep it simple and useful across various editorial pages. I implemented simple "Skip To the main content" as good start. Benefit is also that we have that translation already in the system.

Do you think is this good enough improvement? Or do you think something else should be included?

<img width="695" height="352" alt="Image" src="https://github.com/user-attachments/assets/5693989c-4b75-46b1-abae-79d1a151a3a0" />

**Comment by israelcefrin** [comment]
Hello @jardakotesovec 

I think that including a link to skip to the main content is a great start. I wonder if it would be possible to include a second link to the "Site Navigation" component too. Given that it is already in the markup and is also translated.

**Comment by jardakotesovec** [comment]
@israelcefrin Sounds good, added Site Navigation. And this is now also available on jarda.testing if you would like to give it a try. Thanks!

**Comment by israelcefrin** [comment]
Tested and looks great, thanks @jardakotesovec !

**Comment by Tribunal33** [comment]
@jardakotesovec Looks good.  For requirements clarity when the user selects `Site Navigation` the focus will jump to?
When the user selects 'Skip To the main content` the focus will jump to?

I will assume the requirements are something like this.
- [x] On fresh page load of dashboard the first `TAB` will open up Skip Content options menu.  Essentially the menu is located in the first position after browser controls.
- [x] On subsequent loads of the dashboard, for example after users chooses a different filter, the Skip Content Options do not appear after first `TAB`.
- [x] Choosing between either Skip to option is done with `TAB` not the arrow keys
- [x] The Skip to menu will close once the user presses `TAB` after the last item in the menu.
- [x] If user selects Skip to Main Content will take the user to the Editorial Activity Main content 
- [x] If user selects Skip to Site Navigation the focus will be on the side navigation area

I see a slight issue with the border is that just me?
<img width="260" height="82" alt="Image" src="https://github.com/user-attachments/assets/60cb7829-515d-4579-ae31-70fe20a4517f" />

~- Could we also get the Site Navigation First position and Main Content Second as it would then Skip to sections in a linear way.~
- Could we make the capitalization and text consistent?
- 
Will this be back ported to 3.4 so we can close off #9312?  <-- it was fixed there too.

**Comment by jardakotesovec** [comment]
Merged to 3.5 and main. 
@Tribunal33 feel free to close.

**Comment by Tribunal33** [comment]
Looks good I see all the requirements I put forth have been passed.  I added this to the Accessibility testing regression document.

**Comment by jardakotesovec** [comment]
Just for record, additional improvement was included on 3.5 and main branch, as it was creating incorrect hash in the url. Since the focus is driven programatically I removed the hash altogether so its not clashing with other uses of the hash across ojs.
